napari/_qt/_tests/test_qt_notifications.py

```diff
@@ -181,11 +202,11 @@ def test_notification_error(mock_show, monkeypatch, clean_current):
 
 
 @pytest.mark.sync_only
-def test_notifications_error_with_threading(make_napari_viewer):
+def test_notifications_error_with_threading(make_napari_viewer, clean_current):
     """Test notifications of `threading` threads, using a dask example."""
     random_image = da.random.random((10, 10))
     with notification_manager:
-        viewer = make_napari_viewer()
+        viewer = make_napari_viewer(strict_qt=False)
```

Comment:
I'm not sure about this, but the parent for Dialog is set to canvas and we expect that canvas could leak from time to time, so I prefer workaround and use `clean_current` also here 

https://github.com/napari/napari/blob/d28381d88b8ead6efd0aabb01c1cafc368846a87/napari/utils/_testsupport.py#L245